library/Garp/Test/PHPUnit/Helper.php

```diff
@@ -22,14 +22,33 @@ public function setUp(array &$mockData) {
         Garp_Cache_Manager::purge();
         Garp_Model_Db_BindingManager::destroyAllBindings();
 
-        $this->_insertMockData($mockData);
+        $this->_insertPredefinedMockData($mockData);
     }
 
     public function tearDown(array $mockData) {
```

Comment:
@harmenjanssen Moeten we misschien ook bijhouden welke modellen gebruikt worden als er Faker mockdate wordt ingevoerd? Zodat deze ook tijdens de teardown worden getruncate.